decidim-budgets/app/views/decidim/budgets/projects/show.html.erb

```diff
@@ -27,6 +27,8 @@
         </div>
       </div>
     </div>
+    <%= render partial: "decidim/shared/reference", locals: { subject: project } %>
```

Comment:
I'd create a helper for this instead. The helper might as well render a partial, but I don't see it necessary.